File: JAVA/academic_people/Person.java
package pessoa_academico;

public class Person {
    
    protected int age;
    protected String name;

    public Person(){}
    public Person(String _name){
        this.name = _name;
    }

    public void setName(String _name){
        this.name = _name;
    }
    public void setAge(int _age){
        this.age = _age;
    }
    public String getName(){
        return name;
    }
    public int getAge(){
        return age;
    }
  
}

File: CPP/pessoa_academico/Person.cpp
#include <iostream>
using namespace std;
#include "header.h"

class Person{
    private:
    string name;
    int age;

    Person(){}
    Person(string _name){
        name = _name;
    }

    public:
    void setName(string _name){
        name = _name;
    }
    void setAge(int _age){
        age = _age;
    }
    string getName(){
        return name;
    }
    int getAge(){
        return age;
    }
    
};
